dev/tidb-in-kubernetes/maintain/lightning.md

```diff
@@ -0,0 +1,139 @@
+
+            {{< copyable "" >}}
+
+            ```yaml
+            apiVersion: v1
+            kind: Secret
+            metadata:
+              name: cloud-storage-secret
+            type: Opaque
+            stringData:
+              rclone.conf: |
+              [s3]
+              type = s3
+              provider = AWS
+              env_auth = false
+              access_key_id = <my-access-key>
+              secret_access_key = <my-secret-key>
+              region = us-east-1
+
+              [ceph]
+              type = s3
+              provider = Ceph
+              env_auth = false
+              access_key_id = <my-access-key>
+              secret_access_key = <my-secret-key>
+              endpoint = <ceph-object-store-endpoint>
+              region = :default-placement
+
+              [gcs]
+              type = google cloud storage
+              # The service account must include Storage Object Viewer role
+              # The content can be retrieved by `cat <service-account-file.json> | jq -c .`
+              service_account_credentials = <service-account-json-file-content>
+            ```
+
+            Fill in the placeholders with your configurations and save it as `secret.yaml`, and then create the secret via `kubectl apply -f secret.yaml -n <namespace>`.
+
+        3. Configure the `dataSource.remote.storageClassName` to an existing storage class in the Kubernetes cluster.
+
+2. Deploy TiDB Lightning
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm install pingcap/tidb-lightning --name=<tidb-lightning-release-name> --namespace=<namespace> --set failFast=true -f tidb-lightning-values.yaml --version=<chart-version>
+    ```
+
+When TiDB Lightning fails to restore data, it cannot simply be restarted, but manual intervention is required. So the tidb-lightning's `Job` restart policy is set to `Never`.
+
+> **Note:**
+>
+> Currently, TiDB Lightning will [exit with non-zero error code even when data is successfully restored](https://github.com/pingcap/tidb-lightning/pull/230). This will trigger the job failure. Therefore, the success status needs to be determined by viewing tidb-lightning pod's log.
+
+If the lightning fails to restore data, follow the steps below to do manual intervention:
+
+1. Delete the lightning job by running `kubectl delete job -n <namespace> <tidb-lightning-release-name>-tidb-lightning`.
+
+2. Create the lightning job again with `failFast` disabled by `helm template pingcap/tidb-lightning --name <tidb-lightning-release-name> --set failFast=false -f tidb-lightning-values.yaml | kubectl apply -n <namespace> -f -`.
+
+3. When the lightning pod is running again, use `kubectl exec -it -n <namesapce> <tidb-lightning-pod-name> sh` to exec into the lightning container.
```

Comment:
> exec into the lightning container.

What does this mean?

Reply:
The `kubectl exec` command is common for k8s users, and `exec` is usually used in this scenario. Maybe I should use backquote for `exec` to make it special word.